devito/finite_differences/derivative.py

```diff
@@ -192,22 +195,43 @@ def T(self):
             adjoint = direct
 
         return Derivative(self.expr, *self.dims, deriv_order=self.deriv_order,
-                          fd_order=self.fd_order, side=self.side, stagger=self.stagger,
-                          transpose=adjoint)
+                          fd_order=self.fd_order, side=self.side, transpose=adjoint,
+                          x0=self.x0)
+
+    def eval_at(self, var):
+        """
+        Evaluates the derivative at the location of var. This is necessary for staggered
+        setup where one could have Eq(u(x + h_x/2, v(x).dx)) in which case v(x).dx
```

Comment:
Is that a type? i.e. should it be `Eq(u(x + h_x/2), v(x).dx)`?

Reply:
yes